src/ui/components/search/searchcomponent.js

```diff
@@ -273,6 +273,12 @@ export default class SearchComponent extends Component {
      * @type {string}
      */
     this.inputIdName = `yxt-SearchBar-input--${this.name}`;
+
+    /**
+     * Options to pass to the autocomplete component
+     * @type {Object}
+     */
+    this._autocompleteConfig = config.autocomplete || {};
```

Comment:
what kind of config is being passed? I think if possible we want to avoid passing config directly

Reply:
took a look at the theme prs and I'm back! would it make sense to just expose the config needed for the overlay? we would probably want to add it to the readme as well

Reply:
Ah thanks!! I just updated to pass only the specific new config: onOpen and onClose hooks!